### PR TITLE
Refactor(MessageAttachment): Use `Attachment` instead

### PR DESCRIPTION
**Pull request by Mateo-tem**
**Please describe the changes this PR makes and why it should be merged:**

Changes `MessageAttachment` to `Attachment`, because can be used also on Interactions, not just in Messages. 
Also, `Message` is being deprecated. So, can be a good chance to generalize.

**Status and versioning classification:**

- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating
- This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

File: packages/discord.js/src/client/websocket/WebSocketManager.js [hunk 1]
```diff
@@ -136,7 +136,7 @@ class WebSocketManager extends EventEmitter {
       shards: recommendedShards,
       session_start_limit: sessionStartLimit,
     } = await this.client.rest.get(Routes.gatewayBot()).catch(error => {
-      throw error.httpStatus === 401 ? invalidToken : error;
+      throw error.status === 401 ? invalidToken : error;
```
Comment (imranbarbhuiya) [inline packages/discord.js/src/client/websocket/WebSocketManager.js:139]: Out of scope

**Review by imranbarbhuiya — CHANGES_REQUESTED** [review]

**Comment by BenjammingKirby** [comment]
should this be on the builders package?

**Comment by imranbarbhuiya** [comment]
> should this be on the builders package?

Yeah AttachmentBuilder in builders and Attachment in djs to make consistent with all other builders.

File: packages/discord.js/src/client/websocket/WebSocketManager.js (hunk 1 above)
Comment (ImRodry) [reply packages/discord.js/src/client/websocket/WebSocketManager.js]: I believe this was recently merged so you might just need to rebase

**Comment by imranbarbhuiya** [comment]
You literally deleted the file xD.

**Comment by Mateo-tem** [comment]
I'm new to doing this xD.

**Comment by Syjalo** [comment]
> To be honest, I'm new to doing this xD.

Without `WebSocketManager` discord.js won't work. Return it back.

**Comment by Mateo-tem** [comment]
Done.

File: packages/discord.js/src/client/websocket/WebSocketManager.js [hunk 2]
```diff
@@ -391,4 +391,4 @@ class WebSocketManager extends EventEmitter {
   }
 }
 
-module.exports = WebSocketManager;
+module.exports = WebSocketManager;
```
Comment (ImRodry) [inline packages/discord.js/src/client/websocket/WebSocketManager.js:394]: Undo this

Comment (Syjalo) [reply packages/discord.js/src/client/websocket/WebSocketManager.js:394]: Add a new line

**Review by kyranet — APPROVED** [review]

**Review by bilgehanertan — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Review by SpaceEEC — CHANGES_REQUESTED** [review]
There still is a reference to `MessageAttachment` left:
https://github.com/Mateo-tem/discord.js/blob/a74412bd1e7e759a0c0d1e52b1b7e344eb61a696/packages/discord.js/src/managers/GuildStickerManager.js#L44

Otherwise LGTM

**Review by SpaceEEC — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Pull request merged**